tests/TestZipLineConfig.py

```
@@ -0,0 +1,239 @@
+                with patch.object(
+                    global_config_store,
+                    "get_relative_path",
+                    side_effect=lambda *parts: os.path.join(tmp, *parts),
+                ), patch.object(config_module, "get_relative_path",
+                                side_effect=lambda *parts: os.path.join(tmp, *parts)), \
+                    patch.object(global_config_store, "_MIGRATION_STATE_PATH", state_path), \
+                    patch.object(global_config_store, "_MIGRATION_BACKUP_DIR", backup_dir), \
+                    patch.object(account_scope_store, "_STORE_PATH", store_path):
+                    config = global_config_store.get_global_config(ZIP_LINE_CONFIG_NAME)
+
+                # 全局结果：旧键值复制到新键
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)